stacks-signer/src/main.rs

```diff
@@ -320,12 +320,28 @@ fn handle_generate_stacking_signature(
     )
     .expect("Failed to generate signature");
 
-    if do_print {
-        println!(
-            "\nSigner Public Key: 0x{}\nSigner Key Signature: 0x{}\n\n",
+    let output_str = if args.json {
+        serde_json::to_string_pretty(&serde_json::json!({
```

Comment:
If it's going to be for copy/paste, why not just print the JSON directly?  If I wanted it pretty-printed, I'd pipe to `jq`

Reply:
Do you feel like pretty-printed JSON is harder to copy/paste and/or share via DM? My thought is that they're both easy to handle, while pretty-print by default allows for more easily debugging the output for someone who might not know about `jq`.

Reply:
Pretty printed json is significantly more difficult to copy paste due to the expansion.